newhelm/sut_decorator.py

```diff
@@ -1,16 +1,20 @@
 from functools import wraps
 import inspect
+from typing import Sequence, Type
 from newhelm.record_init import add_initialization_record
 from newhelm.sut import SUT
+from newhelm.sut_capabilities import SUTCapability
 
 
-def newhelm_sut():
+# TODO: Remove defaulting.
+def newhelm_sut(capabilities: Sequence[Type[SUTCapability]] = []):
```

Comment:
A limitation of using the decorator is it means instances of the class must all have the same capabilities. If we think that is an issue, we may need to find a different way to do this.

Reply:
This seems like a minor issue for now. I think that across together, openAI, and huggingface, the only model that doesn't return logprobs is gpt-4-vision-preview in [openAI](https://platform.openai.com/docs/api-reference/chat/create#chat-create-logprobs). But I could see the need for different SUT instances to be able to toggle on/off it's class's default capabilities in the future.